website/content/api-docs/system/lease-count-quotas.mdx

```diff
@@ -26,12 +27,19 @@ that can either be a namespace or mount.
 - `path` `(string: "")` - Path of the mount or namespace to apply the quota.
   A blank path configures a global lease count quota. For example `namespace1/`
```

Comment:
As noted below, I really like the switch of non-root to non-global. It might be beneficial to clarify that "global" indicates that it will exist at the root namespace. Not sure if that is too much though. What are your thoughts?